### PR TITLE
Prep GH Pages for new Jakarta EE 10 release

### PR DESCRIPTION
**Pull request by kwsutter**
Signed-off-by: Kevin Sutter <kwsutter@gmail.com>

Initialize our GH Pages for the Jakarta EE 10 release.

One question is whether we still need or want the Specifications.md page.  It's difficult to maintain (easy to get out of date).  It's also referencing the EE4J implementations, which is a little biased when we have Compatible Implementations for some of the specs outside of the EE4J domain.  Comments or suggestions on this page would be appreciated.

### CODE REVIEW

File: jakartaee9/JakartaEE9.1.md [hunk 1]
```diff
@@ -24,10 +24,9 @@ No other individual component (feature) release plans are expected.
```
Comment (kazumura) [inline jakartaee9/JakartaEE9.1.md:24]: remove "(work in progress)"

File: jakartaee9/JakartaEE9.1.md [hunk 2]
```diff
@@ -24,10 +24,9 @@ No other individual component (feature) release plans are expected.
 
 Overall schedule dates are ***tentative*** and ***subject to change***.
```
Comment (kazumura) [inline jakartaee9/JakartaEE9.1.md:25]: remove this line

**Review by kazumura — COMMENTED** [review]
Thanks, @kwsutter .
I left some minor suggestions for "JakartaEE9.1.md".

**Comment by kazumura** [comment]
Regading Specification.md page, I agree it is difficult to maintain.
And we can get almost same information from "https://jakarta.ee/specifications/".
So I would suggest to remove Specification.md.

**Comment by kwsutter** [comment]
> Regading Specification.md page, I agree it is difficult to maintain.
> And we can get almost same information from "https://jakarta.ee/specifications/".
> So I would suggest to remove Specification.md.

Thanks for the review and "vote" for removal of Specifications.md page, @kazumura!  Barring any further comment to keep it, I will probably remove it.

File: jakartaee9/JakartaEE9.1.md (hunk 1 above)
Comment (kwsutter) [reply jakartaee9/JakartaEE9.1.md:24]: Will do.  Thanks!

File: jakartaee9/JakartaEE9.1.md (hunk 2 above)
Comment (kwsutter) [reply jakartaee9/JakartaEE9.1.md:25]: Good idea.  Thanks!

**Review by ivargrimstad — APPROVED** [review]

**Comment by kwsutter** [comment]
> > Regading Specification.md page, I agree it is difficult to maintain.
> > And we can get almost same information from "https://jakarta.ee/specifications/".
> > So I would suggest to remove Specification.md.
> 
> Thanks for the review and "vote" for removal of Specifications.md page, @kazumura! Barring any further comment to keep it, I will probably remove it.

Decided to go ahead with the removal.  We already have a reference to the main https://jakarta.ee/specifications/ page and that's a better reference anyway.

**Review by kazumura — APPROVED** [review]

**Comment by edbratt** [comment]
Rather than remove it, what about just editing the Specifications page so it requires less maintenance. The links to the GH repositories are harder to find from the Specifications normative pages. We could certainly update the Specification column target to the normative intermediate landing page (i.e. specifications/annotations (as opposed to specifications/annotations/2.0 as an example). I could take a stab at this and if the consensus still is, discard it, we can still do that.

**Comment by kwsutter** [comment]
> Rather than remove it, what about just editing the Specifications page so it requires less maintenance. The links to the GH repositories are harder to find from the Specifications normative pages. We could certainly update the Specification column target to the normative intermediate landing page (i.e. specifications/annotations (as opposed to specifications/annotations/2.0 as an example). I could take a stab at this and if the consensus still is, discard it, we can still do that.

Something along the lines of what I just committed?  Seems like a good compromise.

**Comment by edbratt** [comment]
@kwsutter Yeah, that looks good to me. Hopefully it won't require frequent updates. Thanks!

**Comment by ivargrimstad** [comment]
> @kazumura and @ivargrimstad, since you both had already approved this PR, what do you think about this slightly modified Specifications.md page instead of just deleting it?

I like it!

**Comment by kazumura** [comment]
> @kazumura and @ivargrimstad, since you both had already approved this PR, what do you think about this slightly modified Specifications.md page instead of just deleting it?

Looks better!

**Comment by kwsutter** [comment]
Thanks for the quick reviews!  Per my comments on yesterday's Platform call, I also updated the expected release date for 1Q2022 in the schedule.

**Pull request merged**